draft-irtf-cfrg-vdaf.md

```diff
@@ -3020,8 +3029,21 @@ removing it after unsharding.
 
 # IANA Considerations
 
-This document makes no request of IANA.
+| Value      | Scheme               | Type | Reference |
+|:-----------|:---------------------|:-----|:----------|
+| 0x00000000 | reserved for testing | n/a  | n/a       |
+| 0x00100000 | Prio3Aes128Count     | VDAF | {{prio3aes128count}} |
+| 0x00100001 | Prio3Aes128Sum       | VDAF | {{prio3aes128sum}} |
+| 0x00100002 | Prio3Aes128Histogram | VDAF | {{prio3aes128histogram}} |
+| 0x00200000 | Poplar1Aes128        | VDAF | {{poplar1aes128}} |
+{: #codepoints title="Unique identifers for (V)DAFs."}
 
+> TODO Add IANA considerations for the codepoints summarized in {{codepoints}}.
+
+> OPEN ISSUE Currently the scheme includes the PRG. This means that we need bits
+> of the codepoint to differentiate between PRGs. We could instead make the PRG
+> generic (e.g., Prio3Count(Aes128) instead of Prio3Aes128Count) and define a
+> separate codepoint for.
```

Comment:
I would leave it as-is for now. I don't know see any reason why an application would want to be opinionated about the PRG, so it makes sense to me that we pick one or maybe two variants that work well for all.

Reply:
I think there are a couple potential reasons for this:
1. It's useful to be able to reuse existing primitives whenever possible. (For instance, @simon-friedberger found himself in the position of having to re-implement `PrgAes128` from primitives in NSS.) This suggests to me that there may eventually be a preference to build something from, say, HMAC. (HKDF is not particularly well suited here, given its relatively short output length.)
2. In #32 we're discussing an alternative AES-based PRG. One might have a preference to stick with what we have today. (I hope not, though.)

In any case, we can cross this bridge later.